lib/solvers/LongDistancePairSolver/LongDistancePairSolver.ts

```diff
@@ -0,0 +1,170 @@
+import { getConnectivityMapsFromInputProblem } from "lib/solvers/MspConnectionPairSolver/getConnectivityMapFromInputProblem"
+import type { MspConnectionPair } from "lib/solvers/MspConnectionPairSolver/MspConnectionPairSolver"
+import type {
+  InputProblem,
+  InputPin,
+  PinId,
+  InputChip,
+} from "lib/types/InputProblem"
+import { BaseSolver } from "../BaseSolver/BaseSolver"
+import { getAllPossiblePairs } from "lib/utils/get-all-possible-pairs"
+import { SchematicTraceSingleLineSolver2 } from "../SchematicTraceLinesSolver/SchematicTraceSingleLineSolver2/SchematicTraceSingleLineSolver2"
+import { doesTraceOverlapWithExistingTraces } from "lib/utils/does-trace-overlap-with-existing-traces"
+import { visualizeInputProblem } from "../SchematicTracePipelineSolver/visualizeInputProblem"
+import type { SolvedTracePath } from "../SchematicTraceLinesSolver/SchematicTraceLinesSolver"
+import type { ConnectivityMap } from "connectivity-map"
+
+export class LongDistancePairSolver extends BaseSolver {
```

Comment:
```suggestion
/**
 * Solves for traces that can be connected over a longer-than-acceptable trace distance IF there are
 * no intersections with other traces
 * 
 * We allow these traces to exceed the max length because they have no intersections with other traces
 **/
export class LongDistancePairSolver extends BaseSolver {
```